coalib/output/printers/LogPrinter.py

```diff
@@ -115,7 +115,7 @@ def log_exception(self,
 
         self.log(log_level, message, timestamp=timestamp, **kwargs)
```

Comment:
are there 2 spaces between "It's" and "better"? make it one please.

also, it should be "It's"->"it's".

needs comma after the "-L DEBUG"

enclose "-L DEBUG" in double backticks as its a command.

end body with period.